share/functions/prompt.fish

```diff
@@ -0,0 +1,81 @@
+function prompt
+    argparse h/help -- $argv
+    or return
+
+    set -l cmd $argv[1]
+    set -e argv[1]
+
+    if set -q _flag_help
+        __fish_print_help prompt
+        return 0
+    end
+
+    if contains -- $cmd list; and set -q argv[1]
+        echo "Too many arguments" >&2
+        return 1
+    end
+
+    set -l prompt_dir $__fish_data_dir/sample_prompts $__fish_data_dir/tools/web_config/sample_prompts
```

Comment:
(note: This was added in preparation of moving the sample_prompts directory out of web_config - because it no longer *is* web-exclusive)